cookbooks/bcpc/recipes/cobbler.rb

```diff
@@ -345,3 +350,13 @@
   action [:enable,:start]
   notifies :run, 'bash[cobbler-sync]', :delayed
 end
+
+#
+# After recurring problems with reloading the xinetd
+# configuration, we resort to a forced restart on every chef run.
```

Comment:
Do you have some concrete pointers which can be referenced so we know when this may be pulled out?